src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2096,8 +2096,10 @@ private void InternalSaveAs(string path, SaveContext saveContext, bool isBackup
             {
                 Model.Logger.Log(String.Format(Properties.Resources.SavingInProgress, path));
                 CurrentSpaceViewModel.Save(path, isBackup, Model.EngineController, saveContext);
-
                 if (!isBackup) AddToRecentFiles(path);
+                // TODO: DYN-6050 This is not the final place to check this property, this is only for Testing
+                if (currentWorkspaceViewModel?.IsHomeSpace ?? true)
+                    Model.Logger.Log("The Workspace is valid for FDX : " + Model.CurrentWorkspace.IsValidForFDX);
```

Comment:
Maybe consider to use `CurrentWorkspace.HasRunWithoutCrash && Model.CurrentWorkspace.IsValidForFDX` and remove the other check